dist/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Do-List</title>
+<script defer src="index.bundle.js"></script></head>
+<body>
+  <h1>Today's To Do</h1>
+  <div id="todoList"></div>
```

Comment:
- [ ] 👍 Well done on creating an object and using a function to dynamically add the list of todos! However, visually, your code does not match the design. To achieve a better visual alignment, please follow the design closely and update your code accordingly. Ensure that the font styles, spacing, and overall layout match the design requirements for displaying the list of todos.

| Your | Design |
| --- | --- |
| ![image](https://github.com/Nessrine88/Todolist/assets/55840999/03be36c0-b791-4973-b4f0-2d20ab8be5a5)  | ![image](https://github.com/Nessrine88/Todolist/assets/55840999/801dc715-47db-4989-bd21-364916e47f18) |


By making these updates, your to-do list will look more polished and visually consistent with the design. Keep up the good work, and I'm confident your attention to design details will lead to a more refined project! 🎨📝🚀